examples/resources/runbook/example.tf

```diff
@@ -1,6 +1,7 @@
+# DEPRECATED: Use the `cells` field instead
 resource "shoreline_runbook" "data_runbook" {
   name        = "data_runbook"
-  description = "A sample runbook defined using the data field, which loads the runbook configuration from a separate JSON file."
+  description = "(DEPRECATED)A sample runbook defined using the data field, which loads the runbook configuration from a separate JSON file."
```

Comment:
Why is description deprecated? I think you meant to leave a comment for the `data` field?